src/components/accordion/_accordion.scss

```diff
@@ -1,3 +1,9 @@
+//
```

Comment:
Should be able to use the same docblock comment syntax in scss, right? The `/**` kind

Reply:
yeah I think the transpiled output will preserve the comments if we use that syntax right? Do we want that as well or would it be sufficient to have the header in the source file only

Reply:
I tested and confirmed that the `/** */` comments will remain in the output stylesheet

Reply:
Does the output style from sass impact things if the distributed CSS is minified anyways?

Reply:
I guess not but the unminified one would be larger if you were to build it. for what it's worth it looks like material components uses the `//` comment styles for their SCSS as well, for similar reasons I imagine

Reply:
Sounds good, will make sure to update in elements then 😄